### PR TITLE
Feature/add interaction config

### PR DESCRIPTION
**Pull request by jbauerrfid**
### What changes were proposed in this pull request?

* add configuration for smartcosmos-ext-interactions
* add API link for smartcosmos-ext-interactions

#### Depends On

* https://github.com/SMARTRACTECHNOLOGY/smartcosmos-ext-interactions/pull/8

### CODE REVIEW

File: config/smartcosmos-ext-interactions.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+server:
+  port: 45556
```
Comment (sdomula) [inline config/smartcosmos-ext-interactions.yml]: shouldn't the port be set like this `port: ${smartcosmos.service.ext-interactions.port}`

File: docker-compose.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-version: '2'
+version: '3'
```
Comment (sdomula) [inline docker-compose.yml]: Why do you increase the version here? We would need to port to the new docker-compose file format. Let's do this in a separate PR. :)

**Review by sdomula — CHANGES_REQUESTED** [review]

File: config/smartcosmos-ext-interactions.yml (hunk 1 above)
Comment (andreassiegel) [reply config/smartcosmos-ext-interactions.yml]: yes

File: config/smartcosmos-ext-interactions.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  endpoints:
+    interactions:
+      enabled: true
+      create.enabled: true
+  interactions:
+    kafka:
+      host: ${smartcosmos.service.kafka.address}
+      port: 9092
+      address: ${smartcosmos.service.kafka.host}:${smartcosmos.service.kafka.port}
+
```
Comment (sdomula) [inline config/smartcosmos-ext-interactions.yml]: Remove `host` and `port`.

**Review by sdomula — CHANGES_REQUESTED** [review]

File: config/smartcosmos-ext-interactions.yml (hunk 3 above)
Comment (jbauerrfid) [reply config/smartcosmos-ext-interactions.yml]: updated this also in https://github.com/SMARTRACTECHNOLOGY/smartcosmos-ext-interactions/pull/8

File: docker-compose.yml (hunk 2 above)
Comment (ghost) [reply docker-compose.yml]: I believe this will break docker since this is the version of the compose file.

**Review by ghost — APPROVED** [review]

File: config/smartcosmos-ext-interactions.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  port: ${smartcosmos.service.ext-interactions.port}
+
+smartcosmos:
+  security:
+    enabled: true
+  endpoints:
+    interactions:
+      enabled: true
+      create.enabled: true
+  interactions:
```
Comment (sdomula) [inline config/smartcosmos-ext-interactions.yml:11]: If the above settings are default (they should), remove them from this settings file.

**Review by sdomula — CHANGES_REQUESTED** [review]

**Comment by sdomula** [comment]
I'll close this PR, as it is not relevant anymore.

**Pull request closed without merging**